test/repl/object-ext.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Tests that fields completely override base fields
+moc -i <<__END__
+{ x = -25 in { x = 42 : Nat } };
+
+module X {public let a = 7};
+{ y = -25 in { y = 3; z = "H" } and X };
```

Comment:
Out curiosity, why is this a repl test?

Reply:
Will be rewritten, as it is not testing the compiler.

Reply:
It could be a `test/fail` one

Reply:
I'll leave it for now, it is somewhat informative.